src/main/main.go

```diff
@@ -16,10 +16,20 @@ func main() {
 	config.ProcessArgs()
 	logger := utils.InitLogger(config.GlobalConfig.Verbose)
 	logger.Infof("Assisted installer started. Configuration is:\n %+v", config.GlobalConfig)
+
```

Comment:
OK, I think that since we should try to get the pull secret from env, if for wome reason it's not there we should get it from the file.
In any case I think the logic of obtaining it should move to config / utils / it's own module